src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -484,13 +486,25 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                         id="min"
                         labelText="Min"
                         value={min || ''}
+                        invalid={parseInt(min) > parseInt(max)}
```

Comment:
We don't generally restrict obs to being integers so it may be important not to do so here? (`parseFloat()` vs `parseInt()`.

Reply:
Done!